src/Kentico.Xperience.Lucene/Admin/Client/tsconfig.json

```diff
@@ -14,5 +14,5 @@
     "forceConsistentCasingInFileNames": false,
     "noFallthroughCasesInSwitch": true
   },
-  "include": ["./src/**/*"]
+  "include": [ "src/**/*.ts", "src/**/*.tsx", "dist/*.js", "src/*.tsx" ]
```

Comment:
I don't think we want to include `dist/*` because that is the compiled output of `src/*`

Was there an issue with the previous `include`?

Reply:
I had issues with front end development, but this does not need to be there. Removed.